src/pg/counts/counts-pg-store.ts

```diff
@@ -54,58 +56,76 @@ export class CountsPgStore extends BasePgStoreModule {
 
   async applyInscriptions(writes: DbInscriptionInsert[]): Promise<void> {
     if (writes.length === 0) return;
-    await this.sqlWriteTransaction(async sql => {
-      const mimeType = new Map<string, any>();
-      const rarity = new Map<string, any>();
-      const type = new Map<string, any>();
-      for (const i of writes) {
-        const t = i.number < 0 ? 'cursed' : 'blessed';
-        mimeType.set(i.mime_type, {
-          mime_type: i.mime_type,
-          count: mimeType.get(i.mime_type)?.count ?? 0 + 1,
-        });
-        rarity.set(i.sat_rarity, {
-          sat_rarity: i.sat_rarity,
-          count: rarity.get(i.sat_rarity)?.count ?? 0 + 1,
-        });
-        type.set(t, { type: t, count: type.get(t)?.count ?? 0 + 1 });
-      }
-      await sql`
-        INSERT INTO counts_by_mime_type ${sql([...mimeType.values()])}
+    const mimeType = new Map<string, any>();
```

Comment:
Curious why we are removing the write-transaction?

Reply:
It's not being removed, the diff looks a bit weird but I'm performing all the writes in a single query (using `WITH x AS ()`)

Reply:
Oh I think you're asking about the `writeTransaction`, that's no longer necessary because I only run 1 query now instead of several